src/modules/aws_ecs_fargate@0.0.1/index.ts

```diff
@@ -228,12 +228,12 @@ export const AwsEcsFargateModule: Module = new Module({
     taskDefinition: new Mapper<TaskDefinition>({
       entity: TaskDefinition,
       equals: (a: TaskDefinition, b: TaskDefinition) => Object.is(a.cpuMemory, b.cpuMemory)
-        && Object.is(a.executionRoleArn, b.executionRoleArn)
+        && AwsIamModule.utils.roleEq(a.executionRole, b.executionRole)
```

Comment:
This is ok but checking the `arns` should be enough?

Reply:
actually, the names are guaranteed unique. I did this before realizing that